man/en/sos-collect.1

```diff
@@ -0,0 +1,312 @@
+.TH sos collect 1 "April 2020"
+
+.SH NAME
+sos collect \- Collect sosreports from multiple (cluster) nodes
+.SH SYNOPSIS
+.B sos collect
+    [\-a|\-\-all\-options]
+    [\-b|\-\-become]
+    [\-\-batch]
+    [\-c CLUSTER_OPTIONS]
+    [\-\-chroot CHROOT]
+    [\-\-case\-id CASE_ID]
+    [\-\-cluster\-type CLUSTER_TYPE]
+    [\-e ENABLE_PLUGINS]
+    [--encrypt-key KEY]\fR
+    [--encrypt-pass PASS]\fR
+    [\-\-group GROUP]
+    [\-\-save\-group GROUP]
+    [\-\-insecure-sudo]
+    [\-k PLUGIN_OPTION]
+    [\-\-label LABEL]
+    [\-n SKIP_PLUGINS]
+    [\-\-nodes NODES]
+    [\-\-no\-pkg\-check]
+    [\-\-no\-local]
+    [\-\-master MASTER]
+    [\-o ONLY_PLUGINS]
+    [\-p SSH_PORT]
+    [\-\-password]
+    [\-\-password\-per\-node]
+    [\-\-preset PRESET]
+    [\-s|\-\-sysroot SYSROOT]
+    [\-\-ssh\-user SSH_USER]
+    [\-\-sos-cmd SOS_CMD]
+    [\-t|\-\-threads THREADS]
+    [\-\-timeout TIMEOUT]
+    [\-\-tmp\-dir TMP_DIR]
+    [\-v|\-\-verbose]
+    [\-\-verify]
+    [\-z|\-\-compression-type COMPRESSION_TYPE]
+
+.PP
+.SH DESCRIPTION
+collect is an sos subcommand to collect sosreports from multiple nodes and package
+them in a single useful tar archive. 
+
+sos collect can be run either on a workstation that has SSH key authentication setup
+for the nodes in a given cluster, or from a "master" node in a cluster that has SSH
+keys configured for the other nodes.
+
+Some sosreport options are supported by sos-collect and are passed directly to 
+the sosreport command run on each node.
```

Comment:
Why don't support all `sos report` options? (not necessarily resolved in this PR, for sure)

Reply:
That's a mix of bad wording (should be "applicable sosreport options are supported") and a bit of a hold over from the standalone tool.

At this point, the non-supported options would be `--allow-system-changes`, `--no-env-vars`, `plugin-timeout` (though `-k plugin.timeout` should work), and `--since-`. Easy enough to add those - if you'd prefer I can add them to this patchset.

Reply:
Would it be possible to separate in the help the collector options from the report options?  It would make it a bit cleaner.   *Not a blocker.

Reply:
Currently / in this proposal, `sos report` options are maintained at two places: in `man/en/sos-collect.1` and `man/en/sosreport.1`. Isn't it better to defer here the `sos report` options to man pages of `sosreport`? With the notion "this and that option differs/not supported" (like `-t` applies to threads of `sos-collect`)?